### PR TITLE
feat(test_infra) - added adversarial controls for chunk production

### PR DESCRIPTION
**Pull request by wacban**
* Added a new adversarial control that disables/enables chunk production.
* Added a new rpc endpoint to control the above. 
* Added an example python test using the controls. 

I need to test the scenario where a range of blocks is missing chunks at the end of epoch. This is needed to test the state sync and congestion control integration. I'll do that in a follow up PR.

### CODE REVIEW

**Review by jakmeier — COMMENTED** [review]
Looks good to me but I am vastly unfamiliar with the python testing setup. Let's wait for @VanBarbascu for approval.

**Comment by wacban** [comment]
https://nayduck.nearone.org/#/run/83

**Review by nagisa — APPROVED** [review]

**Pull request merged**
